tests/test_evaluation.py

```diff
@@ -214,24 +213,24 @@ def test_merge_ranking(rating_true, rating_pred):
 
 def test_python_rmse(rating_true, rating_pred):
     assert (
-        rmse(
-            rating_true=rating_true,
-            rating_pred=rating_true,
-            col_prediction=DEFAULT_RATING_COL,
-        )
-        == 0
+            rmse(
+                rating_true=rating_true,
+                rating_pred=rating_true,
+                col_prediction=DEFAULT_RATING_COL,
+            )
+            == 0

```

Comment:
Have you tested it under black formatter? When I run my black formatted, this code block doesn't change like this format.